app/api/groups.py

```diff
@@ -31,6 +32,18 @@ def before_post(self, args, kwargs, data):
         if not current_user.is_verified:
             raise ForbiddenError({'source': ''}, 'Access Forbidden')
```

Comment:
As we are using before_create_object now, move all of it there and remove before_post and delete `data['user'] = current_user.id`

Reply:
Yes :v: